.spelling

```diff
@@ -396,6 +396,7 @@ memcached
 memcached-2's
 Mesos
 mesos-dns
+MeshConfig
```

Comment:
Don't add this to the spelling exceptions. Use back-ticks around `MeshConfig` instead.
```suggestion
```